proposals/lip-0061.md

```diff
@@ -564,14 +571,21 @@ def verifyAggregateCommit(blockHeader: BlockHeader)-> bool:
     # maxHeightPrecommitted before processing blockHeader and updating the BFT store.
     if aggregateCommit.height > getBFTHeights().maxHeightPrecommitted:
         return False
+    # The heights of aggregate commits must be at least MIN_CERTIFY_HEIGHT.
+    if aggregateCommit.height < MIN_CERTIFICATE_HEIGHT:
+        return False
     # If there are new BFT parameters for a height h, then the chain needs to include
     # an aggregate commit for the block at height h-1. This block and the corresponding
     # certificate authenticate the BFT parameters via the validators hash property.
     # Note that the BFT parameters at height maxHeightCertified+1 are already authenticated
     # by the previous certificate.
-    heightNextBFTParameters = getNextHeightBFTParameters(getBFTHeights().maxHeightCertified+1)
-    if previous function call returns valid height and aggregateCommit.height > heightNextBFTParameters-1:
-        return False
+    try:
+        heightNextBFTParameters = getNextHeightBFTParameters(getBFTHeights().maxHeightCertified+1)
```

Comment:
I think it would be better to do
```
heightNextBFTParameters = max(MIN_CERTIFICATE_HEIGHT + 1, getNextHeightBFTParameters(getBFTHeights().maxHeightCertified + 1)
```
here and remove the additional condition `heightNextBFTParameters > MIN_CERTIFICATE_HEIGHT` below. This implies that we enforce that a certificate for height `MIN_CERTIFICATE_HEIGHT` is generated. Note that with the current specs the height of the first certificate can be arbitrarily large for the Lisk Mainchain (because `heightNextBFTParameters > MIN_CERTIFICATE_HEIGHT` will always be `False` as long as no certificate is generated).

Reply:
I switched to your suggestion, meaning we force a certificate for height `MIN_CERTIFICATE_HEIGHT`.